### ISSUE TITLE
Container stats rx_bytes of undefined

### ISSUE THREAD

**Issue by andreas4all** [crosbymichael/dockerui#167]
Hi, I have problem with dockerui stats about containers. It's on all containers.

```
TypeError: Cannot read property 'rx_bytes' of undefined
    at l (dockerui.js:6)
    at dockerui.js:6
    at angular.js:20
    at i (angular.js:7)
    at angular.js:7
    at n.$eval (angular.js:8)
    at n.$digest (angular.js:8)
    at n.$apply (angular.js:8)
    at f (angular.js:7)
    at r (angular.js:7)
```

```
Docker API Version: 1.21 UI Version: v0.8.1
```

Can you please help me?

**Comment by kevana** [comment]
Could you post the output of `docker version` and `docker info`?

**Comment by andreas4all** [comment]
```
$ docker version
Client:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   a34a1d5
 Built:        Fri Nov 20 13:12:04 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   a34a1d5
 Built:        Fri Nov 20 13:12:04 UTC 2015
 OS/Arch:      linux/amd64
```

```
$ docker info
Containers: 61
Images: 1316
Server Version: 1.9.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 1505
 Dirperm1 Supported: false
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 3.13.0-66-generic
Operating System: Ubuntu 14.04.3 LTS
CPUs: 12
Total Memory: 15.67 GiB
Name: servername.localhost
ID: K6FN:7V3D:AJA6:6QC3:XOSC:BLNX:WMCP:AHAI:GCQL:2HKL:Q5NP:HG57
WARNING: No swap limit support
```

**Comment by virtualfunction** [comment]
Yes, this is affecting me on 1.9.1 - it's preventing the stats from updating

**Comment by kevana** [comment]
Fix is in latest `0.9.0-beta` image on the hub. It only grabs stats for the first network but I haven't seen many people using multiple networks yet.

**Comment by virtualfunction** [comment]
Cool, thanks for sorting this :) Just noticed another issue though with slow AJAX queries stopping the stats. I'll open it as another issue in the sec
